.github/file-filters.yml

```diff
@@ -56,15 +56,25 @@ backend: &backend
   - 'migrations_lockfile.txt'
   - 'config/**/*'
 
-plugins:
+acceptance: &acceptance
+  - *backend
+  - *frontend
+
+plugins: &plugins
   - *backend
   - 'src/sentry_plugins/**/*.html'
 
-api_docs:
+api_docs: &api_docs
   - *backend
   - 'api-docs/**'
   - 'tests/apidocs/**'
 
+# This is the ultimate controller for backend.yml
+backend_any_type: &backend_any_type
+  - *backend
+  - *api_docs
+  - *plugins
```

Comment:
In the backend.yml we have jobs that check on either one of these. In order to handle the fake workflow we use this.